rotkehlchen/chain/manager.py

```diff
@@ -52,7 +52,9 @@
 from rotkehlchen.chain.substrate.manager import wait_until_a_node_is_available
 from rotkehlchen.chain.substrate.typing import KusamaAddress
 from rotkehlchen.chain.substrate.utils import KUSAMA_NODE_CONNECTION_TIMEOUT
-from rotkehlchen.constants.assets import A_ADX, A_BTC, A_DAI, A_ETH, A_ETH2, A_KSM
+from rotkehlchen.chain.substrate.typing import PolkadotAddress
```

Comment:
You have two imports from `from rotkehlchen.chain.substrate.typing` that can be combined

```suggestion
from rotkehlchen.chain.substrate.typing import KusamaAddress, PolkadotAddress
```

Reply:
yep that was before I ran the lint. I added some commits